SonarQube.TeamBuild.PreProcessor/Interfaces/IAnalyzerProvider.cs

```diff
@@ -7,6 +7,7 @@
 
 using SonarQube.Common;
 using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.PreProcessor.Roslyn;
```

Comment:
Keeping the comment chain intact - it does appear that we aren't writing "sonar.branch" out, so we should add that.